### ISSUE TITLE
[Feature Request] Upmix stereo to 5.1 or 7.1 DSP

### ISSUE THREAD

**Issue by Alex9001** [fooyin/fooyin#158]
Any chance to get an option to upmix stereo playback to additional speakers?

![image](https://github.com/fooyin/fooyin/assets/2770710/595f7c06-25e4-40a4-9786-8a48e9b7f3fb)

**Comment by ludouzi** [comment]
Yeah, I'll be adding that as a DSP along with DSP chains. It's not a priority right now, but I'll get around to it soon.

**Comment by kode54** [comment]
May I suggest freesurround? It already bundles KissFFT for its operations, but can fairly easily be adapted to different FFT libraries, including a system installed one.
